charts/plgd-hub/values.yaml

```diff
@@ -2016,6 +2016,161 @@ grpcgateway:
           certFile:
           useSystemCAPool: false
 
+grpcreflection:
+  # -- Enable grpc-reflection service
+  enabled: true
+  # -- Name of component. Used in label selectors
+  name: grpc-reflection
+  # -- Full name to override
+  fullnameOverride:
+  # -- Number of replicas
+  replicas: 1
+  # -- Additional labels for grpc-reflection deployment
+  deploymentLabels: {}
+  # -- Additional annotations for grpc-reflection deployment
+  deploymentAnnotations: {}
+  # -- Pod security context
+  podSecurityContext: {}
+  # -- Labels for grpc-reflection pod
+  podLabels: {}
+  # -- Annotations for grpc-reflection pod
+  podAnnotations: {}
+  service:
+    # -- Service type
+    type: ClusterIP
+    # -- Labels for grpc-reflection service
+    labels: {}
+    # -- Annotations for grpc-reflection service
+    annotations: {}
+    # -- Target port
+    targetPort: grpc
+    # -- Protocol
+    protocol: TCP
+    # -- Name
+    name: grpc
+    crt:
+      # -- Extra DNS names for service certificate
+      extraDnsNames: []
+  # -- RBAC configuration
+  rbac:
+    # Enable RBAC
+    enabled: false
+    # -- Name of grpc-reflection SA
+    serviceAccountName: grpc-reflection
+    # -- Template definition for Role/binding etc..
+    roleBindingDefitionTpl:
+  # -- Security context for pod
+  securityContext: {}
+  # -- Image pull secrets
+  imagePullSecrets: {}
+  # -- Restart policy for pod
+  restartPolicy: Always
+  # -- Init containers definition
+  initContainersTpl: {}
+  # -- Extra POD containers
+  extraContainers: {}
+  image:
+    # -- Image registry
+    registry: ghcr.io/
+    # -- Image repository
+    repository: plgd-dev/hub/grpc-reflection
+    # -- Image tag.
+    tag:
+    # -- Image pull policy
+    pullPolicy: Always
+    # -- Image pull secrets
+    imagePullSecrets: {}
+  # -- Liveness probe. grpc-reflection doesn't have any default liveness probe
+  livenessProbe: {}
+  # -- Readiness probe. grpc-reflection doesn't have aby default readiness probe
+  readinessProbe: {}
+  # -- Resources limit
+  resources: {}
+  # -- Node selector
+  nodeSelector: {}
+  # -- Affinity definition
+  affinity: {}
+  # -- Toleration definition
+  tolerations: {}
+  # -- Optional extra volumes
+  extraVolumes: {}
+  # -- Optional extra volume mounts
+  extraVolumeMounts: {}
+  # -- External domain for grpc-reflection. Default: api.{{ global.domain }}
+  domain:
```

Comment:
Reflection services need to be registered for each domain according to domains of services (each can run on different domain or all/some of them can run on same domain):
- grpcgateway.pb.GrpcGateway
- certificateauthority.pb.CertificateAuthority
- snippetservice.pb.SnippetService
- m2moauthserver.pb.M2MOAuthService

so for each domain we need to create deployment and pod with certain reflected services.